Packages/com.unity.inputsystem/Documentation~/Gamepad.md

```diff
@@ -1,21 +1,23 @@
 # Gamepad Support
 
-* [Controls](#controls)
-* [Polling](#polling)
-* [Rumble](#rumble)
-  * [Pausing, resuming, and stopping haptics](#pausing-resuming-and-stopping-haptics)
-* [PlayStation controllers](#playstation-controllers)
-* [Xbox controllers](#xbox-controllers)
-* [Switch controller](#switch-controllers)
-* [Cursor Control](#cursor-control)
+- [Gamepad Support](#gamepad-support)
+  - [Controls](#controls)
+    - [Deadzones](#deadzones)
+  - [Polling](#polling)
+  - [Rumble](#rumble)
+    - [Pausing, resuming, and stopping haptics](#pausing-resuming-and-stopping-haptics)
+  - [PlayStation controllers](#playstation-controllers)
+  - [Xbox controllers](#xbox-controllers)
+  - [Switch controllers](#switch-controllers)
+  - [Cursor Control](#cursor-control)
 
 A [`Gamepad`](../api/UnityEngine.InputSystem.Gamepad.html) is narrowly defined as a Device with two thumbsticks, a D-pad, and four face buttons. Additionally, gamepads usually have two shoulder and two trigger buttons. Most gamepads also have two buttons in the middle.
 
 A gamepad can have additional Controls, such as a gyro, which the Device can expose. However, all gamepads are guaranteed to have at least the minimum set of Controls described above.
 
 Gamepad support guarantees the correct location and functioning of Controls across platforms and hardware. For example, a PS4 DualShock controller layout should look identical regardless of which platform it is supported on. A gamepad's south face button should always be the lowermost face button.
 
->NOTE: Generic [HID](./HID.md) gamepads will __not__ be surfaced as [`Gamepad`](../api/UnityEngine.InputSystem.Gamepad.html) devices but rather be created as generic [joysticks](./Joystick.md). This is because the Input System cannot guarantee correct mapping of buttons and axes on the controller (the information is simply not available at the HID level). Only HID gamepads that are explicitly supported by the Input System (like the PS4 controller) will come out as gamepads. Note that you can set up the same kind of support for specific HID gamepads yourself (see ["Overriding the HID Fallback"](./HID.md#overriding-the-hid-fallback)).
+>NOTE: Generic [HID](./HID.md) gamepads will __not__ be surfaced as [`Gamepad`](../api/UnityEngine.InputSystem.Gamepad.html) devices but rather be created as generic [joysticks](./Joystick.md). This is because the Input System cannot guarantee correct mapping of buttons and axes on the controller (the information is simply not available at the HID level). Only HID gamepads that are explicitly supported by the Input System (like the PS4 controller) will come out as gamepads. Note that you can set up the same kind of support for specific HID gamepads yourself (see ["Overriding the HID Fallback"](./HID.md#creating-a-custom-device-layout)).
```

Comment:
Given the page is renamed should the name 'overriding the HID fallback' be updated too?